SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+Board meetings shall provide at least one week (7 days) notice to all board members and the Steering Committee.
+Notice of meetings shall specify the place, day, and hour of meeting.
+The purpose of the meeting needs not be specified.
+
+* **Board officers**: The officers of the board shall be a board chair, vice-chair, and secretary, all of whom shall be chosen by, and serve at the pleasure of, the board.
+Officers shall serve one year terms (as officers) and can serve consecutive terms.
+
+* **Board Chair**: The board chair shall lead the board in performing its duties and responsibilities, including, if present, leading all meetings of the board.
+
+* **Vice-chair**: In the absence of the board chair, the vice-chair shall perform the duties of the board chair.
+When so acting, the vice-chair shall have all the powers of and be subject to all the restrictions upon the board chair.
+
+* **Secretary**: The secretary shall be responsible for keeping the minutes of all meetings and actions of the board.
+The minutes of each meeting shall state the time and place that it was held, the members present, and such other information as shall be necessary to determine the actions taken.
+The minutes of all board meetings shall be made publicly accessible to the whole SunPy community.
+
+* **Advise and Consent**: The steering committee shall present the state of the project and future plans at meetings of the board.
+The advisory board shall advise and consent on those plans.
+
+
+## The Steering Committee
+The role of the steering committee, which is headed by the Lead Developer, is to manage the day to day operations of the SunPy project.
+The steering committee shall be responsible for the running of the project under advisement of the board.
+This includes:
+
+* holding regular meetings with the developer community (at least monthly)
+* making regular reports to the board (at least quarterly)
+* having ownership privileges to the SunPy GitHub organization
+* creating, maintaining and enforcing the SunPy code of conduct (see below)
+* accepting or rejecting SEPs based on community consensus
+* delegating responsibility and authority to other community roles
+* planning and managing finances and financial processes of the project.
+
+### Composition
+
+The steering committee is formed of three roles:
+
+* **The lead developer** chairs the steering committee and is responsible for holding regular (at least monthly) meetings, and ensuring that the committee is represented at all board meetings.
+* **The deputy lead developer** in the absence of the lead developer, the deputy shall perform all the duties of the lead developer.
+* **The financial officer** the financial officer is primarily responsible for ensuring the health of the project finances and setting up processes for managing expenses and reporting to the board on financial matters.
+
+The steering committee is elected by the **project members**.
```

Comment:
Must the steering committee members originally be project members? I don't see that written anywhere but i may have missed it.

Reply:
I have not stated that either way. Astropy did not make that a requirement, I am open to opinions either way.

Reply:
I don't think we would enforce it, you never know but I would be very surprised if they were not project memebers beforehand.

Reply:
I have for the moment added a comment saying they *do not* need to be project members. I am happy to discuss that though.

Reply:
@ehsteve [says elsewhere](https://github.com/sunpy/sunpy-SEP/pull/62#discussion_r1146573838)

> I disagree with this. I feel like they should have to have been Project Members before being elected to the Steering Committee. The Project Members will elect them and it is best if they know them and have worked with them before than some external person that Project Members may not know at all.